addons/mail/models/mail_message.py

```diff
@@ -107,6 +107,8 @@ def _get_default_author(self):
     message_id = fields.Char('Message-Id', help='Message unique identifier', index=True, readonly=1, copy=False)
     reply_to = fields.Char('Reply-To', help='Reply email address. Setting the reply_to bypasses the automatic thread creation.')
     mail_server_id = fields.Many2one('ir.mail_server', 'Outgoing mail server')
+    moderation_status = fields.Selection([('pending_moderation', 'Pending Moderation'), ('accepted', 'Accepted'), ('rejected', 'Rejected')], string="Status")
```

Comment:
Couldn't we use the same keys as moderation table, aka allow / ban + pending ?

Reply:
I have used the same terminology as in backend and followed the spec.